packages/zowe-explorer/src/globals.ts

```diff
@@ -35,7 +35,7 @@ export let DS_DIR: string;
 export let CONFIG_PATH; // set during activate
 export let ISTHEIA = false; // set during activate
 export let LOG: imperative.Logger;
-export const COMMAND_COUNT = 105;
+export const COMMAND_COUNT = 108;
```

Comment:
This value seems to be only used in `extension.unit.test.ts`. Should we define the constant at the top of that test file rather than here?